.travis/osx..install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-PACKAGES="cmake pkg-config fftw libogg libvorbis libsndfile libsamplerate jack sdl stk portaudio node fltk"
+PACKAGES="cmake pkgconfig fftw libogg libvorbis lame libsndfile libsamplerate jack sdl libgig libsoundio stk portaudio node fltk"
```

Comment:
@zapashcanon thanks for the detailed explanation.  So let's hard-code a `pkgconfig` search as well (right?) to avoid the conflict and then only inject `pkg-config` if neither `pkgconfig` or `pkg-config` is found in the list of installed packages.  Does that make sense?